containers/cloud-streams/Dockerfile

```diff
@@ -36,6 +36,8 @@ RUN $CC_HOME/provision.sh $BASE_TEMPLATE && \
     $CC_HOME/profiles/CCE/data/templates/composite/$BASE_TEMPLATE/test.sh && \
     $CC_HOME/profiles/CCE/data/templates/composite/sag-is-cloudstreams/test.sh && \
     $CC_HOME/cleanup.sh
+    
+RUN rm -rf $SAG_HOME/profiles/CTP
```

Comment:
I should be part of the above command to minimize the size. I'll make then change foryou